definitions/npm/reselect_v3.x.x/test_reselect_3.x.x.js

```diff
@@ -4,7 +4,7 @@ import {
   defaultMemoize,
```

Comment:
please add the test for our bug :) For example:
```javascript
// TEST: Should work when using another selector as functions
const simpleSelector = (state: State): string => "foo";
const resultSelector = (arg: string): string => "foo";

const combinedSelector1 = createSelector(simpleSelector, resultSelector);

const combinedSelector2: (state: State) => string = createSelector(
  combinedSelector1,
  resultSelector
);
// END TEST
```
This test doesn't pass with Flow 0.28 => 0.33 so you'll need to change the lower bound of the old libdef to 0.34.